### ISSUE TITLE
Message Strip accessibility issues

### ISSUE THREAD

**Issue by lormanlau** [SAP/ui5-webcomponents#4749]
### **Feature Request Description**
Using JAWS screenreader: When using the negative message strip, the "negative closable" part of the announcement must be removed as that is not meaningful to screen reader users. Also, the accessible name (aria-label/aria-labelledby) of the "decline" button should be "close message strip" or changeable.

### **Proposed Solution**
Remove announcement of negative closable and added a way to change the buttons accessibility name.

### **Proposed Alternatives**
Have the button's read close message strip as part of the screen reader.

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: SAP SuccessFactors
- Bussiness impact: Medium

**Comment by ivoplashkov** [comment]
Hello @lormanlau,

This issue has been previously addressed and we have prepared a fix [here](https://github.com/SAP/ui5-webcomponents/pull/4423). The fix is part of version 1.1.2 and is available on the master branch, therefore I am assuming that you are using a previous one. Could you please state which version are you using and if the provided fix works for you?

Thank you in advance!
Best regards,
Ivaylo

**Comment by gmkv** [comment]
Closing issue due to inactivity.
